testsuite/run_sets/build_validation_retail.yml

```diff
@@ -0,0 +1,20 @@
+# This file describes the order of features in a Build Validation test suite run.
+
+## NOTE: This stage can't be run in parallel
+
+## Prepare configuration for branch server and terminals
+- features/build_validation/retail/proxy_branch_network.feature
+- features/build_validation/retail/sle11sp3_terminal_prepare.feature
+- features/build_validation/retail/sle12sp5_terminal_prepare.feature
+- features/build_validation/retail/sle15sp3_terminal_prepare.feature
+
+## Build Kiwi images for all retail terminals
+## (expected time of building is around 1 hour and 20 minutes for each terminal)
+- features/build_validation/retail/sle11sp4_buildhost_build_kiwi_image.feature
+- features/build_validation/retail/sle12sp5_buildhost_build_kiwi_image.feature
+- features/build_validation/retail/sle15sp3_buildhost_build_kiwi_image.feature
+
+## Deployment of retail terminals
+- features/build_validation/retail/sle11sp3_terminal_deploy.feature
+- features/build_validation/retail/sle12sp5_terminal_deploy.feature
+- features/build_validation/retail/sle15sp3_terminal_deploy.feature
```

Comment:
We should not be moving all to one YAML file, as otherwise it will be impossible to parallelize.
If we need to first to terminal_prepare, then build_kiwi_image and then terminal_deploy, let's have a YAML file for each set of sequential features, but not group all together in a unique YAML file, otherwise that can't be paralelized.

Reply:
Yes, I need to revert (at least partially) these changes to yaml files. I did not think about parallelization, my bad.

Reply:
reverted.